src/makedocs.jl

```diff
@@ -106,6 +106,9 @@ is enabled by default.
 
 **`sitename`** is displayed in the title bar and/or the navigation menu when applicable.
 
+**`description`** is the site-wide description that displays in page previews and search
+engines. Defaults to "Documentation for \$sitename".
```

Comment:
Since this is specific to the HTML output (?) perhaps it should be passed in `format = HTML(...)` like some of the other HTML-specific options.

Reply:
Agreed. We can always promote it to a `makedocs` argument later, and have `HTML` default to the `makedocs` value.